Packs/Netskope/Integrations/NetskopeAPIv2/NetskopeAPIv2_test.py

```diff
@@ -0,0 +1,556 @@
+            "urls": ["www.google.com", "www.abc.com"],
+            "type": "exact",
+            "json_version": 2,
+            "modify_by": "modify_by",
+            "modify_type": "Edited",
+            "modify_time": "modify_time",
+            "pending": "applied",
+        }
+    ]
+
+    assert get_updated_url_list(mock_response) == expected_output
+
+
+@pytest.mark.parametrize(
+    "args, expected_output",
+    [
+        (
+            {
+                "start_time": "01-07-2023 00:00",
+                "end_time": None,
+                "insertion_start_time": None,
+                "insertion_end_time": None,
+            },
+            (1673049600, 1690891200, None, None),
+        ),
+    ],
+)
+def test_convert_time_args_to_num(args, expected_output):
+    """
+    Scenario: Test the convert_time_args_to_num function.
+
+    Given:
+     - Command arguments containing time-related arguments.
+
+    When:
+     - Calling convert_time_args_to_num function with the command arguments.
+
+    Then:
+     - Ensure the function returns the expected updated time arguments.
+    """
+    from NetskopeAPIv2 import TimeArgs, convert_time_args_to_num
+
+    mock_date = datetime(2023, 8, 1, 12)
+
+    @freezegun.freeze_time(mock_date)
+    def run_test():
+        assert convert_time_args_to_num(args) == TimeArgs(*expected_output)
+
+    run_test()
+
+
+def test_convert_number_to_str_date():
+    """
+    Scenario: Test the convert_number_to_str_date function.
+
+    Given:
+     - A date number.
+
+    When:
+     - Calling convert_number_to_str_date function with the date number.
+
+    Then:
+     - Ensure the function returns the expected date string.
+    """
+    from NetskopeAPIv2 import convert_number_to_str_date
+
+    date_number = 1677756000
+    expected_output = "02-03-2023 11:20"
+
+    assert convert_number_to_str_date(date_number) == expected_output
+
+
+def test_fetch_incidents(client, requests_mock):
+    """
+    Scenario: Fetch incidents.
+    Given:
+     -XSOAR arguments.
+    When:
+     - fetch-incident called.
+
+    Then:
+     - Ensure last run and incident are correct.
+    """
+
+    from NetskopeAPIv2 import fetch_incidents
+
+    alert_mock_response = util_load_json("alerts")
+    requests_mock.get(f"{SERVER_URL}api/v2/events/data/alert", json=alert_mock_response)
+    event_mock_response = util_load_json("events")
+    requests_mock.get(
+        f"{SERVER_URL}api/v2/events/data/application", json=event_mock_response
+    )
+    requests_mock.get(f"{SERVER_URL}api/v2/events/data/page", json=event_mock_response)
+
+    last_run, incidents = fetch_incidents(
+        client,
+        {
+            "max_fetch": 10,
+            "fetch_events": True,
+            "first_fetch": "2023-01-01 15:00",
+            "max_events_fetch": 10,
+            "event_types": ["application", "page"],
+        },
+    )
+
+    assert incidents[0]["name"] == "application-event ID: c4a0ad0684b73c3746e205a0"
```

Comment:
please compare the whole output, rather than just a single field in a single incident.